packages/runner-ct/src/variables.scss

```diff
@@ -1,5 +1,5 @@
-@forward '~@cypress/design-system';
-@use '~@cypress/design-system' as *;
+@forward './legacy-styles';
+@use './legacy-styles' as *;
```

Comment:
Are these actually legacy styles? or just shared styles between the app & dashboard?

Reply:
https://cypressio.slack.com/archives/C011BAPC614/p1651786758401009?thread_ts=1651619895.578469&cid=C011BAPC614